### ISSUE TITLE
a11y issues: missing labels and unfocusable regions

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-ResourceMeta#3]
A few accessibility issues that have been briefly observed: 
- In "Element meta names", all inputs (`.chosen-search-input.default`) are missing labels.
- `ul.chosen-results` are scrollable regions that should be focusable, though they are not currently. A [similar issue has been noted across core for Omeka S](https://github.com/omeka/omeka-s/issues/2335).
